pkgs/by-name/pa/parpd/package.nix

```diff
@@ -0,0 +1,30 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "parpd";
+  version = "2.3.0";
+
+  src = fetchFromGitHub {
+    owner = "NetworkConfiguration";
+    repo = "parpd";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-6qmoAW9jm7xMRHZUMQLpe0N+UeVnQP8dC4+Iq+d5Eaw=";
+  };
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    homepage = "https://roy.marples.name/projects/parpd";
+    changelog = "https://github.com/NetworkConfiguration/parpd/releases/tag/v${finalAttrs.version}";
+    description = "Proxy ARP Daemon that complies with RFC 1027";
+    maintainers = with lib.maintainers; [ bot-wxt1221 ];
```

Comment:
I dont believe we want to have bots as maintainers?

Reply:
Check his bio ;)

Reply:
Yes, I did. But what do I miss? Its great to have a bot making such contributions. But in the end, I dont he would be capability of maintaining the package in the long run or triaging more complicated issues?

I guess we should have a human overseeing the package?

I can volunteer to maintain this package too.

Reply:
Well it says "Bot is not a bot." so he isn't a bot

Reply:
Whoopsie, sorry about that. 😅